dwitter/templates/dweet/dweet.html

```diff
@@ -7,10 +7,18 @@
       background: white;
       }
     </style>
+  <script type="text/javascript" src="/static/libs/gif.js"></script>
     <script>
       var autoplay = getURLParameter('autoplay');
       var lastError = "";
       var playing = !!autoplay;
+    var FPS = 60;
+    
+    var recording = false;
+    var encoder;
+	var gifCanvas = document.createElement("canvas");
```

Comment:
Just can't help pointing it out: You're mixing spaces and tabs here.

Reply:
Ah, sorry, I thought I replaced them.